src/components/Swipeable.tsx

```diff
@@ -102,12 +102,12 @@ export interface SwipeableProps
   /**
    * Called when action panel gets open (either right or left).
    */
-  onSwipeableOpen?: () => void;
+  onSwipeableOpen?: (direction: 'left' | 'right') => void;
```

Comment:
We already have a `Directions` enum, although it also provides `UP` and `DOWN` directions. Do you think using it here is fine or the fact that it has all four directions would be a problem?

Reply:
Swipeable doesn't open to vertical directions so typing it with Directions is wrong here. I'd like to stay with the current typing.

Side note: we can move Directions to FlingGH definition file - from what I saw it's used only there.